src/components/Navbar.tsx

```diff
@@ -7,30 +7,30 @@ export interface NavbarProps {
 export const Navbar = (props: NavbarProps) => {
   return (
     <>
-      <nav className="sticky top-0 z-50 w-full text-stone-500 border-b border-gray-200 bg-amber-50">
-        <div className="flex justify-between h-auto w-full py-5 px-10">
-          <div className="flex flex-row justify-center items-center">
+      <nav className="sticky top-0 z-50 border-b border-brown backdrop-blur-md">
```

Comment:
```suggestion
      <nav className="sticky top-0 z-50 border-b border-brown backdrop-blur-md bg-background-400">
```
Navbar currently has a transparent background. Text shows up underneath it when scrolling down a page. Solve this by adding a background.

Reply:
Update after discussing this with Braedon: I had not realized there was a backdrop blur for the navbar; this feature was working in Chrome and Edge, but not for Firefox until I updated to a more recent version (I was running v99). See https://bugzilla.mozilla.org/show_bug.cgi?id=1578503 which was only fixed in July.